### DISCUSSION TITLE
README correction

### DISCUSSION THREAD

**Discussion by ArjixWasTaken** [MCJack123/craftos2#301] [General]
C# is cross-platform, it does not depend on the platform, that is because just like java, it uses a runtime.
Not like I recommend using C#, since it requires a runtime so it is not standalone (and it uses a lot of memory as well)

I just wanted to point out that C# is not platform-dependent

> 4. It doesn't rely on any single platform I wanted to keep CraftOS-PC Classic's wide compatibility in CraftOS-PC 2. Using other languages such as C# or Swift are platform-dependent and are not guaranteed to work on any platform.

**Comment by ArjixWasTaken** [comment]
https://www.mono-project.com/

**Comment by bananasov** [comment]
.NET Core is cross-platform if that is what would be used instead of C++, if .NET Framework was used it would be platform-dependent.

**Comment by ArjixWasTaken** [comment]
@Autist69420 have you looked at the link I sent? mono is a .NET Framework alternative
you might argue that it is not official since it isnt by microsoft, but microsoft sponsors it, and it is widely used (for example by all unity games that dont use il2cpp)
